### ISSUE TITLE
missing pom file

### ISSUE THREAD

**Issue by jjhesk** [novoda/bintray-release#51]
did i miss the pom file?
from the console:
```
Skipping upload for missing file '/Users/hesk/github/AdvancedMaterialTabs/MaterialTabsModule/build/publications/maven/pom-default.xml'.
```

so i received a message from the bintray while i tried to sync from the jcenter. I got rejected that I did include the pom file in the repo.  I was working before but it is not now. I kept missing the to upload the pom file. why it is so? but i have had the pom.xml file in the library folder. can someone help me??

**Comment by jjhesk** [comment]
Skipping upload for missing file 'F:\backupgithub\ToolBarLib\advancedtoolbar\build\libs\advancedtoolbar-sources.jar'.
Skipping upload for missing file 'F:\backupgithub\ToolBarLib\advancedtoolbar\build\libs\advancedtoolbar-javadoc.jar'.
Skipping upload for missing file 'F:\backupgithub\ToolBarLib\advancedtoolbar\build\publications\maven\pom-default.xml'.

Am I missing something?

**Comment by jjhesk** [comment]
resolved from #19
